src/main/java/com/openlattice/datastore/search/controllers/SearchController.java

```diff
@@ -240,13 +273,12 @@ public DataSearchResult executeAdvancedEntitySetDataQuery(
             @RequestBody AdvancedSearch search ) {
         DataSearchResult dataSearchResult = new DataSearchResult( 0, Lists.newArrayList() );
 
-        Set<Principal> currentPrincipals = Principals.getCurrentPrincipals();
-
         if ( authorizations.checkIfHasPermissions( new AclKey( entitySetId ),
                 Principals.getCurrentPrincipals(),
                 EnumSet.of( Permission.READ ) ) ) {
 
             EntitySet es = edm.getEntitySet( entitySetId );
+
```

Comment:
you can actually just get rid of everything in this function and just call through to `searchEntitySetData` with the `SearchConstraints.advancedSearchConstraints` object below (and the difference in auditing doesn't matter since the SearchType will be recorded)